Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2922,15 +2922,7 @@ extension BrowserViewController: PreferencesObserver {
              Preferences.Shields.fingerprintingProtection.key:
             tabManager.allTabs.forEach { $0.webView?.reload() }
         case Preferences.Privacy.blockAllCookies.key:
-            if Preferences.Privacy.blockAllCookies.value {
-                tabManager.reset()
-            } else {
-                tabManager.allTabs.forEach {
-                    if let url: URL = $0.webView?.url {
-                        $0.loadRequest(PrivilegedRequest(url: url) as URLRequest)
-                    }
-                }
-            }
+            tabManager.reset()
```

Comment:
So the only change here is that we don't reload all tabs on cookies toggle?

Reply:
yeah